server/models/Request.js

```diff
@@ -0,0 +1,56 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+    {
+        user_id: {
+            type: mongoose.Schema.Types.ObjectId,
+            required: true,
+            ref: 'User'
+        },
+        sitter_id: {
+            type: mongoose.Schema.Types.ObjectId,
+            required: true,
+            ref: 'User'
+        },
+        location: {
+            type: String,
+            default: '',
+        },
+        start: {
+            type: Date,
+            required: true,
+            default: Date.now,
+        },
+        end: {
+            type: Date,
+            required: true,
+            default: () => Date.now() + 24*60*60*1000,
+        },
+        animalType: {
+            type: String,
+            enum: ['dog', 'cat', 'other'],
+            required: true,
+        },
+        usefulInfo: {
+            type: String,
+            default: '',
+        },
+        accepted: {
+            type: Boolean,
+            default: False,
+        },
+        declined: {
+            type: Boolean,
+            default: False,
+        },
+        paid : {
+            type: Boolean,
+            default: False,
+        }
+    },
+    {
+        timestamps: true
+    }
```

Comment:
Hey @seblaifa3315, use `camelCase` for your variable names. Use the proper linters setup in your editor to format your code.